### PR TITLE
[agent][mssql] Fix supports_schema for Microsoft SQL

### PR DESCRIPTION
**Pull request by vpoliukh**
Related ticket: [AL-10969](https://anodot.atlassian.net/browse/AL-10969)

### CODE REVIEW

File: agent/src/agent/pipeline/manager.py [hunk 1]
```diff
@@ -18,23 +18,35 @@
 
 
 def supports_schema(pipeline_: Pipeline) -> bool:
-    if isinstance(pipeline_, (TestPipeline, RawPipeline)):
+    if isinstance(pipeline_, RawPipeline):
```
Comment (AntonZelenin) [inline agent/src/agent/pipeline/manager.py:21]: why did you remove TestPipeline?

File: agent/tests/unit/test_supports_schema.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+from agent import source
+from agent.source import Source
+
+
+def test_supports_schema():
+    for source_type in source.types:
+        source_ = Source(f'test_{source_type}', source_type, {})
+        pipeline_ = pipeline.manager.build_test_pipeline(source_)
+        pipeline.manager.supports_schema(pipeline_)
+    assert True
```
Comment (AntonZelenin) [inline agent/tests/unit/test_supports_schema.py:11]: it's not needed

**Review by AntonZelenin — APPROVED** [review]

**Review by dariakharlan — APPROVED** [review]
